sumatra/decorators.py

```diff
@@ -13,6 +13,7 @@ def main(parameters, [other_args...]):
 """
 from __future__ import unicode_literals
 
+from builtins import str
```

Comment:
Very minor issue, but is it necessary to import `str` from builtins? Is it not available by default if it's a builtin?